core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -389,15 +385,15 @@ public List<ImageStream> getImageStreams() {
 
     // StatefulSets
     public StatefulSet createStatefulSet(StatefulSet statefulSet) {
-        return appsAPIGroupClient.statefulSets().resource(statefulSet).create();
+        return OpenShift.get(getConfiguration()).apps().statefulSets().resource(statefulSet).create();
```

Comment:
just `apps().statefulSets().resource(statefulSet).create()` doesn't work? This already runs in the context of `Openshift` instance, no need to create a new instance.